### PR TITLE
Uniformize strategy properties and params

### PR DESCRIPTION
**Pull request by tarcisiozf**
related to: https://app.asana.com/0/1201848935859401/1202396940091398/f

### CODE REVIEW

File: examples/exec.js [hunk 1]
```diff
@@ -43,14 +43,14 @@ const run = () => {
 
     strategy.ws = ws
 
-    const strategyOpts = {
+    const strategyOptions = {
       symbol: SYMBOLS.EOS_USD,
       tf: TIME_FRAMES.ONE_DAY,
       includeTrades: true,
       seedCandleCount: 5000
     }
```
Comment (dmytroshch) [inline examples/exec.js:51]: seems this properties also should be renamed

**Review by dmytroshch — CHANGES_REQUESTED** [review]

**Review by avsek477 — APPROVED** [review]
lgtm

File: examples/exec.js (hunk 1 above)
Comment (tarcisiozf) [reply examples/exec.js]: done [23ba66e](https://github.com/bitfinexcom/bfx-hf-strategy-exec/pull/23/commits/23ba66e3edeba3d87642e835a0b289bf79683d9b)

**Review by avsek477 — APPROVED** [review]

**Review by dmytroshch — APPROVED** [review]

**Review by dmytroshch — CHANGES_REQUESTED** [review]
I can't backtest. Get `invalid allocation (code: 514)`

Снимок экрана от 2022-06-08 12-56-39

**Review by dmytroshch — APPROVED** [review]

**Pull request merged**